packages/amplify_datastore/lib/amplify_datastore_stream_controller.dart

```diff
@@ -101,7 +100,6 @@ _onListen() {
               payload: SubscriptionDataProcessedEvent(
                 event,
                 modelProvider,
-                'subscriptionDataProcessed',
```

Comment:
I don't think we should change this following [this change](https://github.com/aws-amplify/amplify-flutter/pull/1599/files#diff-874f240e9cbaa1db21444ca8b63fbc0280a638aa2b499e20e48833cd539c86b8).

In the serialize data, event name are different between iOS and Android, it's `syncReceived` in iOS, `subscriptionDataProcessed` in Android. 

When we added logic to emit this event into Flutter, we used `subscriptionDataProcessed` for both platform. So we didn't need to check which platform we are in to capture this event.

This change will likely existing DataStore integration tests with cloud sync enabled.

Reply:
It looks the parameter is not being used for anything currently: https://github.com/aws-amplify/amplify-flutter/blob/main/packages/amplify_datastore_plugin_interface/lib/src/types/hub/SubscriptionDataProcessedEvent.dart#L23

Reply:
Ahh sorry, my bad, line 100 is the event name... this change looks good. 👍